static/app/utils/metrics/constants.tsx

```diff
@@ -13,7 +13,7 @@ import {
 
 export const METRICS_DOCS_URL = 'https://docs.sentry.io/product/metrics/';
 
-export const DEFAULT_METRICS_CARDINALITY_LIMIT = 5000;
+export const DEFAULT_METRICS_CARDINALITY_LIMIT = 140000;
```

Comment:
Updated this one to align with the automator option.
Created a follow up ticket to expose the option to the FE:
https://github.com/getsentry/sentry/issues/73865